examples/space-shooter/README.md

```diff
@@ -1 +1,3 @@
 A beautiful, retro side-scrolling shooter where you must defeat incoming enemies with your mecha transforming spaceship. Huge boss included!
+
+Discover how to organize your project so that, even for an ambitious project, you can keep a clean and tidy creation environment! Check out how this template uses behaviors made with love by the community and also its very own function for dedicated logic.
```

Comment:
project twice, maybe replace the beginning by "Discover how to organize your ambitious project, by keeping a clean tidy creating environment!"